packages/forms/src/Components/Repeater.php

```diff
@@ -948,7 +949,10 @@ public function relationship(string | Closure | null $name = null, ?Closure $mod
 
                 $record = $relationship->save($record);
                 $item->model($record)->saveRelationships();
+                $existingRecords->push($record);
             }
+
+            $component->getModelInstance()->setRelation($component->getRelationshipName(), $existingRecords);
```

Comment:
Nit: does this still work with `$component->getRecord()` instead of `getModelInstance()`? Otherwise looks good, thanks!

Reply:
Yes, it still works. I've changed it!